### ISSUE TITLE
[Overview] Chapter 1

### ISSUE THREAD

**Issue by bartvanb** [vantage6/vantage6-workshop#41]
**Desired solution**

From @bartvanb:
- There are still some TODOs in chapter 1
- "There are different ways in which privacy risks can be mitigated. We will focus on the idea of federated analysis." -> maybe very briefly mention other ways with a link for people that are interested?
- I have the feeling that people will be interested in the connection between the introduced techniques and vantage6. I think there should be a stronger link there with chapter 2.
- Key points need to be added

----------------------------------------------------------------

From @frankcorneliusmartin:
* Did we decide not to mention anything about HE?
* Can we make a simple example on how differencing works in the Federated Learning section? For example lets say we have Alice Bob, Clair and Dave in our dataset. 
  * We have a column age. 
  * We have an average algorithm
     * The algorithm can make sub selection of the data based on age. For example `age > 30`.
  * Create a protocol to obtain the age of each individual in the dataset
* I think it would be good to do a group exercise of federated learning on paper. Can we add that as a challenge.

### Overview
- Questions still need to be done

**Problem statement**
* This is why there are privacy regulations in place like [GDPR](https://gdpr-info.eu/) to prohibit easy access to this wealth of data. -> Maybe also mention the [EHDS]([Europese ruimte voor gezondheidsgegevens - Europese Commissie (europa.eu)](https://health.ec.europa.eu/ehealth-digital-health-and-care/european-health-data-space_nl))
* ~~In the first paragraph maybe mention that there are many reasons why one would not want to centralize data (now it only looks like a legal thing). "However, this centralized approach is undesirable due to several organizational, operational, and political challenges , such as loss of data control, logistics of data transmission, and data governance."~~
* I would not call centralized classic. 

**Federated Learning**
* In the second paragraph. I would emphasise that the gradient leakage is one example of a vulnerability. Maybe briefly also mention the others and that there might be still unknown attacks. I don't feel we should explain them in great detail as that would distract from the point of this chapter. We also could consider making this its own subsection. "Risks in using PETs"
	* Reconstruction
	* Differencing
	* Deep Leakage from Gradients (DLG)
	* Generative Adversarial Networks (GAN)
	* Model Inversion
	* Watermark Attacks
* Maybe we should also mention that we can use it for classical statistical models. 

**Secure Multiparty Computation**
* "Only when all parties work together, the original value can be retrieved." this is not true. This is `n-1` parties work together.
* It only works when parties > 2. So maybe we could use that in an example?

**Differential Privacy**
* Now the description is a bit vague. I would mention that in practice this means adding noise to the data whilst maintaining statistical properties when the data set used is large enough

**Blocks upon blocks**
* I like this section. Please also add that differential privacy can be applied to the datasets before they are used by FL, MPC or HE.

**Data partitioning**
* A picture would be really helpful here.
* "Data can even be horizontally and vertically partitioned at the same time. In these cases, it may be necessary to combine multiple techniques." I've never seen this? Could you give an example?

**Technology doesn’t solve everything**
* Maybe add a link to our DPIA?: [Data Protection Impact Assessment (DPIA) (vantage6.ai)](https://vantage6.ai/data-protection-impact-assessment-dpia/)
* I am missing the data harmonization part in this chapter. This has typically be done before we start any analysis. Techniques like OMOP CDM, FHIR using SNOWMED CT or any common data model can be helpful for this purpose.

**Comment by dsmits** [comment]
@bartvanb what do you mean by saying you can use FL for classicic statistical models? I'm currently making the distinction between federated **learning** and **analysis**. Isn't that enough?

**Comment by dsmits** [comment]
@bartvanb I made a separate issue about the #62 but I'm kind of thinking not to do it. This chapter is by no means meant to be comprehensive. I did add a differencing attack in #63 (kind of).

I could add some links so people can read up on other attacks though

**Comment by bartvanb** [comment]
@dsmits I think these were comments from @frankcorneliusmartin (see https://github.com/vantage6/vantage6-workshop/pull/37 for details) - maybe you can discuss what is meant?

(FYI: the list of items above was a concatenation of all findings of Hector, Frank and myself during the full review, combined with everything that was already open issue for this chapter)

**Comment by dsmits** [comment]
Ok I think this is done now.
